src/store/Wallet.ts

```diff
@@ -213,6 +214,11 @@ const walletState: WalletState = {
   transactionCache: {},
   // Subscriptions to websocket channels.
   subscriptions: {},
+  /**
+   * its own value what is true or false doesn't make sense,
+   * but a change in value means that the current wallet has been changed.
```

Comment:
change comment to: `A change in value means some of the current wallet values have changed.`